server.js

```diff
@@ -1,248 +1,229 @@
+        console.log(`\n ${response.newDept} successfully added to database! \n`);
         startApp();
-    })
+      })
+  })
 };
 
-addADepartment = () => {
+function addARole() {
+  connection.query(`SELECT * FROM department;`, (err, res) => {
+    let departments = res.map(department => ({ name: department.department_name, value: department.department_id }));
     inquirer.prompt([
-        {
-        name: 'newDept',
+      {
+        name: 'title',
         type: 'input',
-        message: 'What is the name of the department you want to add?'   
-        }
+        message: 'What is the name of the role you want to add?'
+      },
+      {
+        name: 'salary',
+        type: 'input',
+        message: 'What is the salary of the role you want to add?'
+      },
+      {
+        name: 'deptName',
+        type: 'rawlist',
+        message: 'Which department do you want to add the new role to?',
+        choices: departments
+      },
     ]).then((response) => {
-        connection.query(`INSERT INTO department SET ?`, 
+      connection.query(`INSERT INTO role SET ?`,
         {
-            department_name: response.newDept,
+          title: response.title,
+          salary: response.salary,
+          department_id: response.deptName,
         },
         (err, res) => {
-            if (err) throw err;
-            console.log(`\n ${response.newDept} successfully added to database! \n`);
-            startApp();
+          if (err) throw err;
+          console.log(`\n ${response.title} successfully added to database! \n`);
+          startApp();
         })
     })
+  })
 };
 
-addARole = () => {
+function addAnEmployee() {
+  connection.query(`SELECT * FROM role;`, (err, res) => {
+    if (err) throw err;
+    let roles = res.map(role => ({ name: role.title, value: role.role_id }));
     connection.query(`SELECT * FROM department;`, (err, res) => {
-        let departments = res.map(department => ({name: department.department_name, value: department.department_id }));
+      if (err) throw err;
+      let departments = res.map(department => ({ name: department.department_name, value: department.department_id }));
+      connection.query(`SELECT * FROM employee;`, (err, res) => {
+        if (err) throw err;
+        let employees = res.map(employee => ({ name: employee.first_name + ' ' + employee.last_name, value: employee.employee_id }));
         inquirer.prompt([
+          createPrompt('firstName', 'input', "What is the new employee's first name?"),
+          createPrompt('lastName', 'input', "What is the employee's last name?"),
+          createPrompt('role', 'rawlist', "What is the employee's title?", roles),
+          createPrompt('dept', 'rawlist', 'What department is the new employee in?', departments),
+          createPrompt('salary', 'number', "What is the new employee's salary?"),
+          createPrompt('manager', 'rawlist', "Who is the new employee's manager?", employees)
```

Comment:
@samrogers15 Regardless of your preference of single vs. double quotes in JS, there's no rule that says you can't use both. Even if you prefer single quotes, I would recommend using double in cases like this where you'd otherwise have to escape characters. IMO, it's easier to read.